### PR TITLE
Fix C-click and option to skip auto-blocking

### PR DESCRIPTION
**Pull request by glogiotatidis**
@ScottDowne r?

### CODE REVIEW

File: campaigns/mofo-eoy-2016/snippet.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
   @blockable: checkbox - Check to allow user to block this snippet.
   @block_button_text - small text - Text to display while hovering over opt out button. Default "Remove this"
-
+  @do_not_autoblock - Check to prevent auto-blocking when user takes action.
```
Comment (glogiotatidis) [inline campaigns/mofo-eoy-2016/snippet.html]: Used inverted / negative variable name to set the default to False and also match this in the admin interface.

File: campaigns/mofo-eoy-2016/snippet.html [hunk 2]
```diff
@@ -479,16 +479,22 @@
...
                 event.stopPropagation();
-                btnDonate.setAttribute("disabled", "disabled");
                 addToBlockList();
+                btnDonate.removeEventListener("click", buttonClickEvent);
                 // Give some time to IndexedDB. Bug 1236090
                 setTimeout(function() { btnDonate.click() }, 500);
-            }
-        });
+             }
+             else {
```
Comment (glogiotatidis) [inline campaigns/mofo-eoy-2016/snippet.html]: We still want to auto-block when user opens a new tab

File: campaigns/mofo-eoy-2016/snippet.html (hunk 1 above)
Comment (ScottDowne) [inline campaigns/mofo-eoy-2016/snippet.html]: Good call.

File: campaigns/mofo-eoy-2016/snippet.html (hunk 2 above)
Comment (ScottDowne) [inline campaigns/mofo-eoy-2016/snippet.html]: Agreed!

**Comment by ScottDowne** [comment]
Thanks for this. I think this looks good. R+.

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**